app/src/main/java/com/checkin/app/checkin/utility/StrikeSpan.kt

```diff
@@ -0,0 +1,40 @@
+package com.checkin.app.checkin.utility
+
+import android.text.SpannableString
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.style.RelativeSizeSpan
+import android.text.style.StrikethroughSpan
+import android.widget.TextView
+import com.checkin.app.checkin.home.epoxy.InvoiceClosedOrderModelHolder
+import com.checkin.app.checkin.session.activesession.InvoiceOrdersAdapter
+
+
+fun InvoiceOrdersAdapter.StrikethroughSpaninvoice(originalCost:String,discountcost:String): CharSequence? {
```

Comment:
Since why did function name is named similarly to class names. Again, use camelCase

Reply:
And don't call it discountcost anywhere.. Call it actualCost

Reply:
And frankly, there is no point of extension function here.. Since it has just one line of Code